Dockerfile

```diff
@@ -40,5 +40,11 @@ RUN cp -r /opt/app/rel/logflare/bin/priv/static /opt/app/rel/logflare/lib/logfla
 RUN rm -r /opt/app/rel/logflare/bin/priv
 
 WORKDIR /opt/app/rel/logflare/bin
+
+# expose specific ports, because for some reason
+# when using docker api with supabase-cli, port bindings don't work without exposing them first
+EXPOSE 4000
+EXPOSE 4002
```

Comment:
we should be able to link them with port mapping from docker vs exposing another port 🤔

Reply:
I've added in replication steps, let me know if you can replicate it.